src/docker.cc

```diff
@@ -187,4 +198,17 @@ json::value DockerReader::QueryDocker(const std::string& path) const
   }
 }
 
+void DockerUpdater::start() {
+  if (!reader_.IsConfigured()) {
```

Comment:
Let's define a virtual `VerifyConfiguration` function in `PollingMetadataUpdater`, make it return `true` by default, call it from `start` in the superclass, and override it here. We might also have to give each updater object a name, so we can identify it in the logs.

Reply:
Updated.